redmine/issue.rb

```diff
@@ -35,17 +35,23 @@ def set_status(status)
     self
   end
 
-  def pull_request
+  def pull_requests
     field = @raw_data['issue']['custom_fields'].find { |f| f['id'] == 7 }
     return nil if field.nil?
     field['value']
   end
 
-  def set_pull_request(url)
+  def set_pull_requests(url)
     @raw_data['issue']['custom_field_values'] = {'7' => url}
     self
   end
 
+  def add_pull_request(url)
+    current_pull_requests = pull_requests
+    current_pull_requests << url
```

Comment:
Yea I can do that -- worth noting in my testing is that Redmine appeared to detect the same value and prevent this sort of duplication at the model level.